docs/build.sbt

```diff
@@ -8,8 +8,8 @@ organizationHomepage := Some(new java.net.URL("http://www.cibotechnologies.com")
 micrositeGithubOwner := "cibotech"
 micrositeGithubRepo := "evilplot"
 micrositeFooterText := None
-micrositeDocumentationUrl := "/cibotech/evilplot/scaladoc/jvm/index.html"
-micrositeBaseUrl := "/cibotech/evilplot/"
+micrositeDocumentationUrl := "/evilplot/scaladoc/jvm/index.html"
+micrositeBaseUrl := "/evilplot/"
```

Comment:
I think removing the trailing `/` here will remove the double `//` showing up in URLs, e.g., 'https://cibotech.github.io/evilplot//drawing-api.html'